### ISSUE TITLE
error running timer 'ac-update-greedy'

### ISSUE THREAD

**Issue by LuoPX-15510486969** [senny/emacs-eclim#226]
I  use emacs-eclipse with prelude and emacs-eclim version is emacs-eclim-20150805.639 which is auto installed.After i lanuch the eclimed and enter the .java file it always show the error ,please look at the screenshot and i do not know what happened . My os is windows7 64 bit.
My confingure file in ~/.emacs.d/personal:
![ecl](https://cloud.githubusercontent.com/assets/13256245/9100271/89f46138-3c0b-11e5-8300-8c7d304e6c86.png)
My problem:
![first-launch](https://cloud.githubusercontent.com/assets/13256245/9100214/dca36e02-3c0a-11e5-9a16-2da1e71bd017.png)
![first](https://cloud.githubusercontent.com/assets/13256245/9100218/e7718580-3c0a-11e5-9059-6829db662fb0.png)

**Comment by gopar** [comment]
Hmm are you calling `start-eclimd` explicitly (or calling `eclim-mode`)? If you just navigate to the java project and try to get completions then it won't work (because eclimd hasn't been started). Thats the only thing I can think of, plus the `C-c C-e is undefined` gives an idea that this might be the problem. 

Also setting `eclim-auto-save` to nil will force you to save the buffer first before being able to retrieve completions. I din't know this until I stopped getting completion and figured that was the issue

**Comment by LuoPX-15510486969** [comment]
yes,all i post is after i call start-eclimd and eclied seems work correctly .

**Comment by darkpssngr** [comment]
I'm getting the same error as well.

**Comment by nloyola** [comment]
Development on this project has moved to https://github.com/emacs-eclim/emacs-eclim. Does it happen with the newer version in that repository?
